Dockerfile

```diff
@@ -21,10 +19,13 @@ COPY config* /var/www/inferno/
 COPY Rakefile /var/www/inferno/
 COPY test /var/www/inferno/test
 COPY lib /var/www/inferno/lib
+COPY db /var/www/inferno/db
+COPY bin /var/www/inferno/bin
+COPY db/config.docker.yml db/config.yml
```

Comment:
Does this need to copy to `/var/www/inferno/db/config.yml`?

Reply:
I actually don't think so since we've set the workdir to `var/www/inferno`, but I'll check.

Reply:
True. It does seem to work for me without that absolute path.